main.go

```diff
@@ -490,7 +497,10 @@ func run(args []string) error {
 			return err
 		}
 
-		status := newStatusHandler(dial)
+		// NOTE: We don't provide a statusTargetAddress here because this the client
+		// /_status endpoint and therefore, its target will be a ghostunnel in server
+		// mode and therefore, should be a (default) TCP check.
+		status := newStatusHandler(dial, "")
```

Comment:
I think that's a reasonable choice - in the future we might want to support this as I know folks are using client mode with non-ghostunnel targets as well where a status might actually be useful.